library/nameops.mli

```diff
@@ -50,16 +50,57 @@ val increment_subscript : Id.t -> Id.t
 
 val forget_subscript    : Id.t -> Id.t
 
+module Name : sig
+
+  include module type of struct include Names.Name end
```

Comment:
@skyskimmer: I don't know. I vaguely imitated code seen elsewhere but my understanding of module type inclusion in Ocaml is not good enough to tell if this is the best way to do. If what you're proposing has the same effect, I have no objections.